ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -63,6 +64,7 @@ object Config {
       participants = Vector.empty,
       maxInboundMessageSize = DefaultMaxInboundMessageSize,
       eventsPageSize = IndexConfiguration.DefaultEventsPageSize,
+      eventsProcessingParallelism = IndexConfiguration.DefaultEventsProcessingParallelism,
```

Comment:
This case class has reached 20 fields this way. Can we group these arguments, please?
For example, the following could be clearly separate groups:
* feature flags
* caches
* metrics
* engine